data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -0,0 +1,44 @@
+---
+datasets:
+  - long_name: "FEMA National Risk Index"
+    short_name: "nri"
+    module_name: national_risk_index
+    last_updated_year: 2020
+    description: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    input_geoid_tract_field_name: "TRACTFIPS"
+    load_fields:
+      - short_name: "ex_loss"
+        df_field_name: "RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME"
+        long_name: "FEMA Risk Index Expected Annual Loss Score"
+        field_type: float
+        tile_include: true
```

Comment:
As discussed in the design discussion in `YAML-Based Score Datasets`, I don't this this is the right place for this field. Whether or not a field should be included on the tile, csv, and excel files is dependent on the score that is currently active. For instance, if Score N is active, it will have certain fields in the tiles that are not there when Score M is active.

Reply:
> For the "include in CSV/Tiles/Excel" prompt, whether or not they should be included will actually depend on what score is "active" in the tool. 
> 
> So maybe the data model is something like: 
> 
> load_fields is a list, and within load_fields, there's a dictionary of {"score_name": {"tile_include": bool, "csv_download": bool, "excel_download": bool}}. 
> 
> E.g., 
> 
> {"Definition M": {"tile_include": True, "csv_download": True, "excel_download": True},
> "Definition N": {"tile_include": True, "csv_download": False, "excel_download": False}}

Reply:
Per our 1x1 conversation, I don't agree with having score definitions on the codebase. I will be working on a score versioning doc, but what I told you on the call, summarized, is that score versions will be tied to Git tags, side effects and static assets will be tied to S3 folders and we will have an upgrade/downgrade score version GHA that will automatically generate these artifacts and also inform the frontend on a version list that could power a pull down on the frontend to select and visualize the different versions.,

Reply:
As discussed, fine with versioning approach, look forward to seeing the design doc and tickets for versioning.

Reply:
TODO: Versioning Doc